docs/dev_guide.md

```diff
@@ -1,31 +1,166 @@
 # Dev Guide
 
-## Adding a new param in GCSFuse
+## Adding new features to GCSFuse
+
+**If you're interested in adding a new feature, please open a GitHub issue to
```

Comment:
If you are interested in contributing a new feature, enhancement, or bug fix, please open